examples/less_basic_spawn.lua

```diff
@@ -0,0 +1,18 @@
+--less_basic_spawn.lua
+local cosock = require "cosock"
+
+local function clock_part(name, sleep_first)
+    return function()
+        if sleep_first then
+            cosock.socket.sleep(1)
+        end
+        while true do
+            print(cosock.socket.gettime(), name)
+            cosock.socket.sleep(2)
+        end
+    end
+end
```

Comment:
This should just be in lined into each call to `spawn` talking about the returning of a function is a bit wordy and off point.